### PR TITLE
chore: add links, maintainers and warn parts to CSV

### PR DESCRIPTION
**Pull request by MatousJobanek**
add parts to CSV:
* links
* maintainers
* warning 
* slightly modified installation description

### CODE REVIEW

File: hack/deploy_csv.yaml [hunk 1]
```diff
@@ -250,7 +250,8 @@ data:
           * Red Hat CodeReady Workspaces
           * OpenShift Pipelines
           ## Installation
-          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and the developer tools get installed in a single namespace.
+          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-che` namespace.
```
Comment (alexeykazakov) [inline hack/deploy_csv.yaml]:
```suggestion
          CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-workspaces` namespace.
```

File: hack/deploy_csv.yaml [hunk 2]
```diff
@@ -250,7 +250,8 @@ data:
...
+          ###### _Warning: This operator is not supposed to be used in OKD - by installing it there you will get unexpected errors. Please, use with OpenShift only._
```
Comment (alexeykazakov) [inline hack/deploy_csv.yaml]:
```suggestion
          ###### _Warning: This operator should be installed in OpenShift Container Platform clusters only. Installing on OKD or other platforms will result in errors._
```

File: deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml [hunk 3]
```diff
@@ -58,7 +58,8 @@ spec:
     * Red Hat CodeReady Workspaces
     * OpenShift Pipelines
     ## Installation
-    CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and the developer tools get installed in a single namespace.
+    CodeReady Toolchain Operator is a cluster-wide operator. It installs and configures the operators for the bundled developer tools to be available across all namespaces in the cluster. CodeReady Toolchain Operator and OpenShift Pipelines get installed in single `openshift-operators` namespace and Red Hat CodeReady Workspaces in single `toolchain-che` namespace.
```
Comment (xcoulon) [inline deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml]: Great! Thanks for taking care of this. @alexeykazakov is it what you request?

File: hack/deploy_csv.yaml (hunk 1 above)
Comment (xcoulon) [reply hack/deploy_csv.yaml]: @alexeykazakov I'm afraid we still use the `toolchain-che` project...

File: deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml (hunk 3 above)
Comment (xcoulon) [reply deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml]: @alexeykazakov same here as below: for now, we create a `toolchain-che` namespace for the Che-related OperatorGroup and Subscription resources.

File: hack/deploy_csv.yaml (hunk 1 above)
Comment (xcoulon) [reply hack/deploy_csv.yaml]: let me open a PR to change that

File: deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml (hunk 3 above)
Comment (alexeykazakov) [reply deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml]: Yeah I know. But we need to fix it. IMO it's OK if we fix the namespace itself in the next PR.

Comment (xcoulon) [reply deploy/olm-catalog/codeready-toolchain-operator/0.0.1/codeready-toolchain-operator.v0.0.1.clusterserviceversion.yaml]: I've opened https://github.com/codeready-toolchain/toolchain-operator/pull/35 for that purpose

File: hack/deploy_csv.yaml (hunk 1 above)
Comment (MatousJobanek) [reply hack/deploy_csv.yaml]: @alexeykazakov sorry, but I missed your comments/suggestions - I changed it here: d58c62c

File: hack/deploy_csv.yaml (hunk 2 above)
Comment (MatousJobanek) [reply hack/deploy_csv.yaml]: same as the previous case - changed before checking your comment d58c62c

**Pull request merged**